tests/PathTest.php

```diff
@@ -823,6 +823,39 @@ public function provideMakeRelativeTests()
             array('phar://C:/webmozart/css/style.css', 'phar://C:/webmozart/puli', '../css/style.css'),
             array('phar://C:/css/style.css', 'phar://C:/webmozart/puli', '../../css/style.css'),
 
+            // already relative + already in root basepath
+            array('../style.css', '/', 'style.css'),
+            array('./style.css', '/', 'style.css'),
+            array('../../style.css', '/', 'style.css'),
+            array('../style.css', 'C:\\', 'style.css'),
```

Comment:
I think the test cases with backslashes should also have backslashes on the left side.